include/phasar/PhasarLLVM/DataFlowSolver/IfdsIde/Solver/IDESolver.h

```diff
@@ -225,6 +252,47 @@ class IDESolver
     return result;
   }
 
+  ///
+  /// Returns the data-flow results at the given statement while respecting
+  /// LLVM's SSA semantics.
+  ///
+  /// An example: when a value is loaded and the location loaded from, here
+  /// variable 'i', is a data-flow fact that holds, then the loaded value '%0'
+  /// will usually be generated and also holds. However, due to the underlying
+  /// theory (and respective implementation) this load instruction causes the
+  /// loaded value to be generated and thus, it will be valid only AFTER the
+  /// load instruction, i.e., at the successor instruction.
+  ///
+  ///   %0 = load i32, i32* %i, align 4
+  ///
+  /// This result accessor function returns the results at the successor
+  /// instruction(s) reflecting that the expression on the left-hand side holds
+  /// if the expression on the right-hand side holds.
+  ///
+  template <typename NTy = n_t>
+  [[nodiscard]]
+  typename std::enable_if_t<std::is_same_v<NTy, const llvm::Instruction *>,
+                            std::unordered_map<d_t, l_t>>
+  resultsAtInLLVMSSA(NTy stmt, bool stripZero = false) {
+    std::unordered_map<d_t, l_t> result = [&valtabe, stmt]() {
+      if (stmt->getType()->isVoidTy()) {
+        return valtab.row(stmt);
+      } else {
+        return valtab.row(stmt->getNextNode());
+      }
+    }();
+    if (stripZero) {
+      for (auto it = result.begin(); it != result.end();) {
+        if (IDEProblem.isZeroValue(it->first)) {
+          it = result.erase(it);
+        } else {
+          ++it;
+        }
+      }
```

Comment:
```suggestion
      // TODO: replace with std::erase_if (C++20)
      for (auto it = result.begin(); it != result.end();) {
        if (IDEProblem.isZeroValue(it->first)) {
          it = result.erase(it);
        } else {
          ++it;
        }
      }
```
Just putting a TODO here so we find this spot when we move at some point to `C++20`